### PR TITLE
Replace string colors with a Color class

### PR DESCRIPTION
**Pull request by roscale**
In order to provide more type safety, the user has to construct Color instances instead of passing strings. Any attempt to construct an invalid color will trigger an assertion.

### CODE REVIEW

**Comment by roscale** [comment]
I'll wait for #44 to be merged, because I'll have to modify the `clear_screen` method too.

Other than that, what do you think about having a Color class? I was thinking that it's probably easier to have colors stored in a friendlier format for eventual color manipulations, or maybe the user wants to provide colors in a different format than RGBA, in which case having a class is better, similar to what we have in Flutter.

**Comment by roscale** [comment]
> I'm quite surprised that there is no actual standard in Python to manage colors.

Same. But what can you do.

File: pygyw/layout/helpers.py [hunk 1]
```diff
@@ -197,16 +199,16 @@ def justify(text: str, width: int):
...
 
-    alpha = int(color[0:2], 16).to_bytes(1, "little")
-    red = int(color[2:4], 16).to_bytes(1, "little")
-    green = int(color[4:6], 16).to_bytes(1, "little")
-    blue = int(color[6:8], 16).to_bytes(1, "little")
+    red = color.red.to_bytes(1, "little")
+    green = color.green.to_bytes(1, "little")
+    blue = color.blue.to_bytes(1, "little")
+    alpha = color.alpha.to_bytes(1, "little")
     return red + green + blue + alpha
```
Comment (TheToto42) [inline pygyw/layout/helpers.py:213]: Shouldn't we move that to the `Color` object ?

File: pygyw/layout/helpers.py [hunk 2]
```diff
@@ -117,7 +118,8 @@ def bottom_text(lines: list, font: fonts.GYWFont, line_height=2.0) -> int:
     """
 
     lines_count = len(lines)
-    return int(settings.screen_height - settings.vertical_padding - lines_count * font.height - (line_height - 1) * font.height * (lines_count - 1))
+    return int(settings.screen_height - settings.vertical_padding - lines_count * font.height - (
+                line_height - 1) * font.height * (lines_count - 1))
```
Comment (TheToto42) [inline pygyw/layout/helpers.py:122]: You can keep the previous version to prevent merge conflicts

File: pygyw/layout/helpers.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 from . import settings
 from . import fonts
+from ..color import Color
```
Comment (TheToto42) [inline pygyw/layout/helpers.py:5]: Why is color there above ? Shouldn't it be at the layout level ?

Comment (roscale) [reply pygyw/layout/helpers.py]: Forgot to remove the import. It was unused.
Do you want me to move color.py inside layout?

Comment (TheToto42) [reply pygyw/layout/helpers.py]: Yes, I would prefer that

**Pull request merged**